### PR TITLE
Add new tokens

### PR DESCRIPTION
**Pull request by vovac12**

### CODE REVIEW

**Comment by Alexey-N-Chernyshov** [comment]
Should Chameleon token be added? I think we come up with name `$KARMA` for it.

**Review by Alexey-N-Chernyshov — COMMENTED** [review]
Please, add $SB and $KARMA tokens.

File: runtime/src/migrations.rs [hunk 1]
```diff
@@ -28,11 +28,44 @@
 // STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 // USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-#[cfg(feature = "ready-to-test")] // kensetsu
 use crate::*;
 
 #[cfg(not(feature = "ready-to-test"))]
-pub type Migrations = ();
+pub type Migrations = (
+    assets::migration::register_asset::RegisterAsset<
```
Comment (Alexey-N-Chernyshov) [inline runtime/src/migrations.rs:35]: If we don't want to release with Kensetsu Phase 1, it is better to remove here.

File: runtime/src/migrations.rs [hunk 2]
```diff
@@ -35,4 +35,51 @@ use crate::*;
 pub type Migrations = ();
 
 #[cfg(feature = "ready-to-test")] // kensetsu
-pub type Migrations = (kensetsu::migrations::remove_hard_cap::RemoveHardCap<Runtime>,);
+pub type Migrations = (
+    kensetsu::migrations::remove_hard_cap::RemoveHardCap<Runtime>,
```
Comment (Alexey-N-Chernyshov) [inline runtime/src/migrations.rs:39]:
```suggestion
```
kensetsu migration `remove_hard_cap` has been applied

File: runtime/src/migrations.rs [hunk 3]
```diff
@@ -35,4 +35,50 @@ use crate::*;
...
+    assets::migration::register_asset::RegisterAsset<
+        Runtime,
+        KARMAAssetId,
+        KARMAAssetName,
+        KARMAAssetSymbol,
+        PredefinedAssetOwnerAccountId,
+    >,
+);
+
+#[cfg(feature = "ready-to-test")] // kensetsu
```
[the remark this answers is not in the document]
Comment (wer1st) [reply runtime/src/migrations.rs:62]: @Alexey-N-Chernyshov ?

[the remark this answers is not in the document]
Comment (Alexey-N-Chernyshov) [reply runtime/src/migrations.rs:62]: No, with release 3.4.0 we have to update migration.

**Review by wer1st — APPROVED** [review]

**Pull request merged**